sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIService.scala

```diff
@@ -57,7 +59,24 @@ private[hive] class SparkSQLCLIService(hiveServer: HiveServer2, sqlContext: SQLC
         case e @ (_: IOException | _: LoginException) =>
           throw new ServiceException("Unable to login to kerberos with given principal/keytab", e)
       }
-    }
+
+    // Also try creating a UGI object for the SPNego principal
```

Comment:
Indentation here is wrong. Also, following blocks have code indented with 4 spaces instead of 2, and wrong indentation.